### DISCUSSION TITLE
Subscriptions

### DISCUSSION THREAD

**Discussion by rogerg1967** [openstore-ecommerce/OpenStore#238] [General]
Hi,
I am looking for a way to allow customers to purchase a subscription to download products. Is this possible?
Thanks
Roger

**Comment by DNNMonster** [comment]
It would be something that you would have to create a plugin for.  I'm not aware of anything that's already available.  To achieve subscriptions one can just create roles for users based upon certain purchase types within a plugin event handler.  Then you can use the roles to expose features as you like.  I think this may also be the best route to take for downloadable files.   I should get corrected if there is an easier pathway.

**Comment by FrancSimari** [comment]
Roger, If I may ask, what have you done to handle subscriptions? Thanks FS
